datanommer.commands/datanommer/commands/__init__.py

```diff
@@ -13,164 +13,30 @@
 #
 # You should have received a copy of the GNU General Public License along
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
+import logging
 import time
 from datetime import datetime, timedelta
 
+import click
 import fedmsg.meta
 from fedmsg.commands import BaseCommand
 from fedmsg.encoding import pretty_dumps
+from fedora_messaging import config as fedora_messaging_config
 from sqlalchemy import func
 
 import datanommer.models as m
 
 
-class CreateCommand(BaseCommand):
-    """Create a database and tables for 'datanommer.sqlalchemy.url'"""
-
-    name = "datanommer-create-db"
-
-    def run(self):
-        m.init(self.config["datanommer.sqlalchemy.url"], create=True)
-
-
-class DumpCommand(BaseCommand):
-    """Dump the contents of the datanommer database as JSON.
-
-    You can also specify a timespan with the --since and --before arguments:
-
-        $ datanommer-dump --before 2013-02-15 --since 2013-02-11T08:00:00 > datanommer-dump.json
-    """
+log = logging.getLogger("datanommer")
 
-    name = "datanommer-dump"
-    extra_args = extra_args = [
-        (
-            ["--since"],
-            {
-                "dest": "since",
-                "default": None,
-                "help": "Only after datetime, ex 2013-02-14T08:05:59.87",
-            },
-        ),
-        (
-            ["--before"],
-            {
-                "dest": "before",
-                "default": None,
-                "help": "Only before datetime, ex 2013-02-14T08:05:59.87",
-            },
-        ),
+try:
+    datanommer_sqlalchemy_url = fedora_messaging_config.conf["consumer_config"][
```

Comment:
I'd rather make this check at the beginning of the commands, this way we can mock the configuration more easily in tests.